docs/src/operators/derivative_operators.md

```diff
@@ -91,11 +91,11 @@ The constructors are as follows:
 
 ```julia
 CenteredDifference{N}(derivative_order::Int,
-                      approximation_order::Int, dx,
+                      approximation_order::Int, dx::Float,
```

Comment:
That doesn't necessarily need to be the case though. It depends on the values and such, and even then it would be `AbstractFloat`. But that's too restrictive to be true.

Reply:
OK, I see. It's just that the error code is pretty misleading. It would be pretty easy to spot with a clearer message.

Reply:
I agree the error message could be better, but I think we need to also enforce correctness.

Reply:
Right, I'll close this one and I'll make another one with a meaningful comment in order to avoid confusion regarding this.